config/ccms/attribute_block_configs/non_passported.yml

```diff
@@ -2576,29 +2576,29 @@ vehicles:
     response_type: text
     generate_block?: true
   CARANDVEH_INPUT_B_14WP2_28A:
-    value: '#vehicle_used_regularly?'
+    value: false
     br100_meaning: 'Cars: It is used for regular use?'
     user_defined: true
     response_type: boolean
-    generate_block?: '#applicant_has_vehicle?'
+    generate_block?: true
   CARANDVEH_INPUT_C_14WP2_25A:
-    value: '#vehicle_estimated_value'
+    value: '0'
     br100_meaning: 'Cars: Current market value'
     user_defined: true
     response_type: currency
-    generate_block?: '#applicant_has_vehicle?'
+    generate_block?: true
   CARANDVEH_INPUT_C_14WP2_26A:
-    value: '#vehicle_payment_remaining'
+    value: '0'
     br100_meaning: 'Cars: Value of loan outstanding'
     user_defined: true
     response_type: currency
-    generate_block?: '#applicant_has_vehicle?'
+    generate_block?: true
   CARANDVEH_INPUT_D_14WP2_27A:
-    value: '#vehicle_purchased_on'
+    value: 01-01-1970
     br100_meaning: 'Cars: Date of purchase'
     user_defined: true
     response_type: date
-    generate_block?: '#applicant_has_vehicle?'
+    generate_block?: true
```

Comment:
probably a wider discussion on the CCMS injection aspect generally possible but just wondering whether we need to be generating the blocks at all. if we are never intending to send vehicle means AND the blocks are not needed by CCMS then we may be able to  set to `false`. 

As it stands i think you can:
1. remove the `generate_block?` line as I think it defaults to `true` 

```
      def generate_attribute_block?(config)
        config.key?(:generate_block?) ? evaluate_generate_block_method(config) : true
      end
```

2. promote to the "superclass" yml config file `base.yml` if it is generally applicable...but not sure

Reply:
This was changing as the PR progressed - it was an early commit so I will review and see what's changed :+1:

Reply:
as discussed happy to rip this all out entirely and see if anything breaks via Mikes UAT and overnight pushing of this data into CCMS EBS....more tech debt ticket/epic to write to investigate whether ripping out virtually everything for means and merits might also work generally